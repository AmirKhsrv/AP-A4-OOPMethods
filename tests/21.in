best_reviewer
recommend_first_approach 129
best_reviewer
recommend_first_approach 65
recommend_first_approach 65
best_reviewer
show_sorted_shelf 112 want_to_read Romance
recommend_second_approach 115
