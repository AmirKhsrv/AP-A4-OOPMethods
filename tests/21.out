id: 31
Name: Athena Souri
Place of Birth: Switzerland
Member Since: December 24 2010
Favorite Genres: Romance
Favorite Authors: Carl Sagan, Colleen Clayton, Greer Noble
Number of Books in Read Shelf: 8
Number of Books in Want to Read Shelf: 15
Number of Books in Currently Reading Shelf: 6
Number of Likes: 2455
id: 357
Title: Dragon Rider
Genre: Fantasy
Author: Cornelia Funke
Reviews:
id: 237 Rating: 1 Likes: 0 Date: March 19 2019
id: 807 Rating: 0 Likes: 0 Date: August 25 2016
id: 1205 Rating: 5 Likes: 0 Date: March 14 2018
id: 2621 Rating: 4 Likes: 0 Date: July 14 2017
id: 2748 Rating: 0 Likes: 0 Date: March 9 2016
id: 3126 Rating: 1 Likes: 0 Date: September 6 2017
id: 3394 Rating: 2 Likes: 0 Date: February 21 2016
id: 3594 Rating: 5 Likes: 0 Date: July 26 2018
id: 3689 Rating: 2 Likes: 0 Date: September 19 2016
id: 3908 Rating: 4 Likes: 0 Date: August 29 2019
id: 4156 Rating: 2 Likes: 10 Date: November 19 2017
id: 4336 Rating: 0 Likes: 52 Date: March 13 2019
id: 4661 Rating: 3 Likes: 0 Date: May 4 2015
id: 5057 Rating: 4 Likes: 0 Date: August 28 2019
id: 6502 Rating: 4 Likes: 5 Date: August 1 2016
id: 6702 Rating: 1 Likes: 0 Date: January 22 2017
id: 6756 Rating: 5 Likes: 6 Date: January 24 2018
id: 6811 Rating: 2 Likes: 0 Date: September 1 2019
id: 6824 Rating: 4 Likes: 10 Date: July 10 2016
id: 8132 Rating: 4 Likes: 0 Date: August 14 2015
id: 8520 Rating: 0 Likes: 0 Date: October 2 2017
id: 8965 Rating: 3 Likes: 0 Date: May 17 2018
id: 8986 Rating: 1 Likes: 0 Date: November 18 2016
id: 9012 Rating: 3 Likes: 0 Date: May 16 2019
id: 9278 Rating: 2 Likes: 146 Date: February 29 2013
id: 9309 Rating: 4 Likes: 0 Date: January 10 2017
id: 9758 Rating: 4 Likes: 0 Date: April 13 2018
id: 10411 Rating: 5 Likes: 0 Date: January 13 2019
id: 11475 Rating: 3 Likes: 0 Date: November 16 2019
id: 12798 Rating: 4 Likes: 0 Date: February 24 2019
id: 12866 Rating: 5 Likes: 0 Date: May 26 2014
id: 13003 Rating: 2 Likes: 0 Date: September 20 2015
id: 13316 Rating: 4 Likes: 0 Date: August 15 2016
id: 13330 Rating: 0 Likes: 1 Date: June 18 2015
id: 13589 Rating: 2 Likes: 0 Date: November 11 2015
id: 14238 Rating: 4 Likes: 4 Date: January 16 2012
id: 14588 Rating: 0 Likes: 0 Date: July 2 2016
id: 14619 Rating: 0 Likes: 0 Date: December 9 2019
id: 14678 Rating: 3 Likes: 1 Date: January 18 2019
id: 14985 Rating: 0 Likes: 0 Date: October 14 2019
id: 15249 Rating: 3 Likes: 0 Date: December 29 2019
id: 15753 Rating: 0 Likes: 1 Date: April 22 2018
id: 16044 Rating: 1 Likes: 1 Date: March 17 2017
id: 16148 Rating: 4 Likes: 0 Date: July 16 2019
id: 16478 Rating: 5 Likes: 0 Date: February 26 2014
id: 16649 Rating: 2 Likes: 0 Date: October 8 2016
id: 16992 Rating: 5 Likes: 0 Date: August 1 2016
id: 17039 Rating: 4 Likes: 0 Date: March 6 2019
id: 17101 Rating: 5 Likes: 0 Date: January 14 2016
id: 17406 Rating: 5 Likes: 0 Date: March 21 2014
id: 31
Name: Athena Souri
Place of Birth: Switzerland
Member Since: December 24 2010
Favorite Genres: Romance
Favorite Authors: Carl Sagan, Colleen Clayton, Greer Noble
Number of Books in Read Shelf: 8
Number of Books in Want to Read Shelf: 15
Number of Books in Currently Reading Shelf: 6
Number of Likes: 2455
id: 357
Title: Dragon Rider
Genre: Fantasy
Author: Cornelia Funke
Reviews:
id: 237 Rating: 1 Likes: 0 Date: March 19 2019
id: 807 Rating: 0 Likes: 0 Date: August 25 2016
id: 1205 Rating: 5 Likes: 0 Date: March 14 2018
id: 2621 Rating: 4 Likes: 0 Date: July 14 2017
id: 2748 Rating: 0 Likes: 0 Date: March 9 2016
id: 3126 Rating: 1 Likes: 0 Date: September 6 2017
id: 3394 Rating: 2 Likes: 0 Date: February 21 2016
id: 3594 Rating: 5 Likes: 0 Date: July 26 2018
id: 3689 Rating: 2 Likes: 0 Date: September 19 2016
id: 3908 Rating: 4 Likes: 0 Date: August 29 2019
id: 4156 Rating: 2 Likes: 10 Date: November 19 2017
id: 4336 Rating: 0 Likes: 52 Date: March 13 2019
id: 4661 Rating: 3 Likes: 0 Date: May 4 2015
id: 5057 Rating: 4 Likes: 0 Date: August 28 2019
id: 6502 Rating: 4 Likes: 5 Date: August 1 2016
id: 6702 Rating: 1 Likes: 0 Date: January 22 2017
id: 6756 Rating: 5 Likes: 6 Date: January 24 2018
id: 6811 Rating: 2 Likes: 0 Date: September 1 2019
id: 6824 Rating: 4 Likes: 10 Date: July 10 2016
id: 8132 Rating: 4 Likes: 0 Date: August 14 2015
id: 8520 Rating: 0 Likes: 0 Date: October 2 2017
id: 8965 Rating: 3 Likes: 0 Date: May 17 2018
id: 8986 Rating: 1 Likes: 0 Date: November 18 2016
id: 9012 Rating: 3 Likes: 0 Date: May 16 2019
id: 9278 Rating: 2 Likes: 146 Date: February 29 2013
id: 9309 Rating: 4 Likes: 0 Date: January 10 2017
id: 9758 Rating: 4 Likes: 0 Date: April 13 2018
id: 10411 Rating: 5 Likes: 0 Date: January 13 2019
id: 11475 Rating: 3 Likes: 0 Date: November 16 2019
id: 12798 Rating: 4 Likes: 0 Date: February 24 2019
id: 12866 Rating: 5 Likes: 0 Date: May 26 2014
id: 13003 Rating: 2 Likes: 0 Date: September 20 2015
id: 13316 Rating: 4 Likes: 0 Date: August 15 2016
id: 13330 Rating: 0 Likes: 1 Date: June 18 2015
id: 13589 Rating: 2 Likes: 0 Date: November 11 2015
id: 14238 Rating: 4 Likes: 4 Date: January 16 2012
id: 14588 Rating: 0 Likes: 0 Date: July 2 2016
id: 14619 Rating: 0 Likes: 0 Date: December 9 2019
id: 14678 Rating: 3 Likes: 1 Date: January 18 2019
id: 14985 Rating: 0 Likes: 0 Date: October 14 2019
id: 15249 Rating: 3 Likes: 0 Date: December 29 2019
id: 15753 Rating: 0 Likes: 1 Date: April 22 2018
id: 16044 Rating: 1 Likes: 1 Date: March 17 2017
id: 16148 Rating: 4 Likes: 0 Date: July 16 2019
id: 16478 Rating: 5 Likes: 0 Date: February 26 2014
id: 16649 Rating: 2 Likes: 0 Date: October 8 2016
id: 16992 Rating: 5 Likes: 0 Date: August 1 2016
id: 17039 Rating: 4 Likes: 0 Date: March 6 2019
id: 17101 Rating: 5 Likes: 0 Date: January 14 2016
id: 17406 Rating: 5 Likes: 0 Date: March 21 2014
id: 357
Title: Dragon Rider
Genre: Fantasy
Author: Cornelia Funke
Reviews:
id: 237 Rating: 1 Likes: 0 Date: March 19 2019
id: 807 Rating: 0 Likes: 0 Date: August 25 2016
id: 1205 Rating: 5 Likes: 0 Date: March 14 2018
id: 2621 Rating: 4 Likes: 0 Date: July 14 2017
id: 2748 Rating: 0 Likes: 0 Date: March 9 2016
id: 3126 Rating: 1 Likes: 0 Date: September 6 2017
id: 3394 Rating: 2 Likes: 0 Date: February 21 2016
id: 3594 Rating: 5 Likes: 0 Date: July 26 2018
id: 3689 Rating: 2 Likes: 0 Date: September 19 2016
id: 3908 Rating: 4 Likes: 0 Date: August 29 2019
id: 4156 Rating: 2 Likes: 10 Date: November 19 2017
id: 4336 Rating: 0 Likes: 52 Date: March 13 2019
id: 4661 Rating: 3 Likes: 0 Date: May 4 2015
id: 5057 Rating: 4 Likes: 0 Date: August 28 2019
id: 6502 Rating: 4 Likes: 5 Date: August 1 2016
id: 6702 Rating: 1 Likes: 0 Date: January 22 2017
id: 6756 Rating: 5 Likes: 6 Date: January 24 2018
id: 6811 Rating: 2 Likes: 0 Date: September 1 2019
id: 6824 Rating: 4 Likes: 10 Date: July 10 2016
id: 8132 Rating: 4 Likes: 0 Date: August 14 2015
id: 8520 Rating: 0 Likes: 0 Date: October 2 2017
id: 8965 Rating: 3 Likes: 0 Date: May 17 2018
id: 8986 Rating: 1 Likes: 0 Date: November 18 2016
id: 9012 Rating: 3 Likes: 0 Date: May 16 2019
id: 9278 Rating: 2 Likes: 146 Date: February 29 2013
id: 9309 Rating: 4 Likes: 0 Date: January 10 2017
id: 9758 Rating: 4 Likes: 0 Date: April 13 2018
id: 10411 Rating: 5 Likes: 0 Date: January 13 2019
id: 11475 Rating: 3 Likes: 0 Date: November 16 2019
id: 12798 Rating: 4 Likes: 0 Date: February 24 2019
id: 12866 Rating: 5 Likes: 0 Date: May 26 2014
id: 13003 Rating: 2 Likes: 0 Date: September 20 2015
id: 13316 Rating: 4 Likes: 0 Date: August 15 2016
id: 13330 Rating: 0 Likes: 1 Date: June 18 2015
id: 13589 Rating: 2 Likes: 0 Date: November 11 2015
id: 14238 Rating: 4 Likes: 4 Date: January 16 2012
id: 14588 Rating: 0 Likes: 0 Date: July 2 2016
id: 14619 Rating: 0 Likes: 0 Date: December 9 2019
id: 14678 Rating: 3 Likes: 1 Date: January 18 2019
id: 14985 Rating: 0 Likes: 0 Date: October 14 2019
id: 15249 Rating: 3 Likes: 0 Date: December 29 2019
id: 15753 Rating: 0 Likes: 1 Date: April 22 2018
id: 16044 Rating: 1 Likes: 1 Date: March 17 2017
id: 16148 Rating: 4 Likes: 0 Date: July 16 2019
id: 16478 Rating: 5 Likes: 0 Date: February 26 2014
id: 16649 Rating: 2 Likes: 0 Date: October 8 2016
id: 16992 Rating: 5 Likes: 0 Date: August 1 2016
id: 17039 Rating: 4 Likes: 0 Date: March 6 2019
id: 17101 Rating: 5 Likes: 0 Date: January 14 2016
id: 17406 Rating: 5 Likes: 0 Date: March 21 2014
id: 31
Name: Athena Souri
Place of Birth: Switzerland
Member Since: December 24 2010
Favorite Genres: Romance
Favorite Authors: Carl Sagan, Colleen Clayton, Greer Noble
Number of Books in Read Shelf: 8
Number of Books in Want to Read Shelf: 15
Number of Books in Currently Reading Shelf: 6
Number of Likes: 2455
id: 302
Title: The Bird Day Battalion (Dom and Kate #1)
Genre: Romance
Author: Genevieve Dewey
***
id: 235
Title: India Black and the Gentleman Thief
Genre: Mystery
Author: Carol K. Carr
***
id: 403
Title: Magic Triumphs
Genre: Fantasy
Author: Ilona Andrews
***
id: 351
Title: Middlemarch
Genre: Classics
Author: George Eliot
***
id: 184
Title: Never Deal with a Dragon
Genre: Fantasy
Author: Robert N. Charrette
***
id: 120
Title: Number 13
Genre: Fantasy
Author: M.G. Wells
***
id: 394
Title: Radio Shangri-La: What I Learned in the Happiest Kingdom on Earth
Genre: Travel
Author: Lisa Napoli
***
id: 260
Title: The Dry
Genre: Mystery
Author: Jane Harper
***
id: 216
Title: The King's Blood
Genre: Fantasy
Author: Daniel Abraham
***
id: 63
Title: When We Wake
Genre: Young Adult
Author: Karen Healey
***
id: 98
Title: Winnie-the-Pooh
Genre: Classics
Author: A.A. Milne
***
id: 211
Title: The Fall of Five
Genre: Young Adult
Author: Pittacus Lore
Reviews:
id: 76 Rating: 5 Likes: 0 Date: August 19 2016
id: 490 Rating: 4 Likes: 0 Date: January 26 2018
id: 1006 Rating: 1 Likes: 3 Date: January 25 2017
id: 1861 Rating: 4 Likes: 0 Date: February 9 2018
id: 2686 Rating: 0 Likes: 0 Date: September 11 2016
id: 2727 Rating: 5 Likes: 0 Date: November 26 2015
id: 3688 Rating: 2 Likes: 0 Date: September 9 2017
id: 3734 Rating: 1 Likes: 0 Date: April 15 2012
id: 4304 Rating: 3 Likes: 26 Date: September 21 2011
id: 4478 Rating: 4 Likes: 1 Date: March 27 2019
id: 4593 Rating: 5 Likes: 0 Date: January 16 2015
id: 4677 Rating: 4 Likes: 1 Date: September 1 2016
id: 4857 Rating: 5 Likes: 4 Date: September 19 2017
id: 5397 Rating: 5 Likes: 0 Date: October 10 2018
id: 6087 Rating: 1 Likes: 0 Date: June 9 2011
id: 6101 Rating: 4 Likes: 0 Date: March 9 2019
id: 7230 Rating: 1 Likes: 10 Date: December 29 2017
id: 7660 Rating: 5 Likes: 3 Date: April 24 2018
id: 8166 Rating: 5 Likes: 0 Date: January 6 2015
id: 8699 Rating: 1 Likes: 0 Date: July 25 2019
id: 9124 Rating: 0 Likes: 0 Date: February 9 2016
id: 9144 Rating: 3 Likes: 0 Date: December 15 2017
id: 9730 Rating: 1 Likes: 0 Date: June 9 2011
id: 9892 Rating: 2 Likes: 0 Date: May 27 2017
id: 10053 Rating: 5 Likes: 0 Date: October 12 2018
id: 10413 Rating: 5 Likes: 0 Date: December 14 2018
id: 10697 Rating: 0 Likes: 22 Date: April 26 2017
id: 10906 Rating: 0 Likes: 0 Date: October 8 2014
id: 10993 Rating: 1 Likes: 0 Date: October 20 2018
id: 11111 Rating: 2 Likes: 3 Date: February 28 2016
id: 11330 Rating: 0 Likes: 26 Date: September 29 2014
id: 11761 Rating: 5 Likes: 0 Date: April 24 2015
id: 11958 Rating: 3 Likes: 0 Date: October 23 2014
id: 12401 Rating: 1 Likes: 0 Date: August 13 2015
id: 12622 Rating: 0 Likes: 0 Date: November 12 2019
id: 13928 Rating: 2 Likes: 0 Date: November 29 2019
id: 14234 Rating: 2 Likes: 2 Date: March 15 2019
id: 14242 Rating: 1 Likes: 1 Date: April 4 2019
id: 14568 Rating: 0 Likes: 0 Date: February 27 2018
id: 16016 Rating: 3 Likes: 0 Date: December 20 2015
id: 16289 Rating: 4 Likes: 0 Date: August 14 2015
id: 16570 Rating: 4 Likes: 0 Date: October 13 2018
id: 16628 Rating: 1 Likes: 0 Date: January 10 2016
id: 16765 Rating: 4 Likes: 0 Date: October 27 2018
